docs/client_quickstart.rst

```diff
@@ -327,7 +327,7 @@ for supported format information.
 Streaming uploads
 =================
 
-:mod:`aiohttp` supports multiple types of streaming uploads, which allows you to
+:mod:`~aiohttp.web` supports multiple types of streaming uploads, which allows you to
```

Comment:
I don't think this is the right solution:
1) This will render as a link with an anchor `web` but the original one was `aiohttp`. To keep this, you could make it explicit:
```suggestion
:mod:`aiohttp <aiohttp.web>` supports multiple types of streaming uploads, which allows you to
```
2) I'm not sure that the original intent was to link `aiohttp.web` module. It looks like it was to refer to the `aiohttp` package.

The problem is that the module is not documented. So I feel like it should be documented and then, it could be linked. But meanwhile, it could be added to the ignore list as shown in https://github.com/aio-libs/aiohttp/pull/5579 and dealt with later, after all the existing references are corrected.

Reply:
Or maybe add `.. currentmodule:: aiohttp` at the top of `index.rst` instead...

Reply:
Alright, I will add it to the ignore list and change it to the way it was `~aiohttp.web`.